spec/factories/posts.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :post do
+    user
+    section
```

Comment:
Oh wow.  This must be a new feature of FactoryGirl I wasn't aware of.  Looks perfect then.  :+1: to keep it like this.